tests/common.sh

```diff
@@ -97,6 +103,9 @@ function install_ide()
     mkdir esp8266com
     cd esp8266com
     ln -s $core_path esp8266
+    # Set custom warnings for all builds (i.e. could add -Wextra at some point)
```

Comment:
The last part of the comment seems to be outdated as `-Werror` is already present.

Reply:
Actually, "-Werror" is included, but the comment says "-Wextra" :)  So this is really OK.

Reply:
I'll stop reviewing PRs until I wake up. Sorry.